src/services/positionService.ts

```diff
@@ -163,8 +163,16 @@ export default class PositionService {
         );
       })
     );
-
-    const results = await Promise.all(promises.map((promise) => promise.catch(() => ({ data: null, error: true }))));
+    const results = await Promise.all(
+      promises.map(async (promise, index) => {
+        const isLastVersion = networksAndVersions[index].version === POSITIONS_VERSIONS[POSITIONS_VERSIONS.length - 1];
```

Comment:
There is a constant that you can use `LATEST_VERSION`

Reply:
Awesome! Thanks